index.html

```diff
@@ -1,6 +1,10 @@
 <!DOCTYPE html>
 <html>
     <head>
+        <meta content="Hour of Code Minecraft!" property="og:title">
+        <meta content="Unnoficial form and mod loader for Minecraft Hour of Code!" property="og:description">
```

Comment:
I wouldn't use HoC or MC here, since these are trademarks... Maybe just `Craft` as title and `["Fun", maybe] fork[/mod, maybe] of Code.org's craft` as description?

Reply:
Sure, edits by maintainers are turned on